gobcore/model/pydantic.py

```diff
@@ -11,3 +12,6 @@ class Schema(BaseModel):
     datasetId: str
     tableId: str
     version: str
+
+    """Required when 'identifier' in Amsterdam Schema is of type list. Ignored otherwise """
```

Comment:
```suggestion
    # Required when 'identifier' in Amsterdam Schema is of type list. Ignored otherwise
```

Or move to class docstring.